packages/core/datadog-sourcemaps.gradle

```diff
@@ -121,34 +119,50 @@ private def getBundleFileResolver(String[] jsBundleDirs, String bundleAssetName)
  *   (and then "$buildDir/generated/assets/createBundle${targetPath}JsAndAssets" with the release of Android Gradle Plugin 7.4, see https://github.com/facebook/react-native/issues/35439)
  * - config was in "project.react" and is now in "project.extensions.react"
  * - accessing parameters values requires calling a getter 
+ * - target path changed from variant.dirName to variant.name
  */
-private def getReactConfig(File buildDir, String targetName, String targetPath) {
+private def getReactConfig(File buildDir, variant) {
     def reactConfig = [:]
 
     if (project.extensions.findByName("react")) {
         // From RN 0.71
+        // Locations and names are defined in "node_modules/react-native-gradle-plugin/src/main/kotlin/com/facebook/react/TaskConfiguration.kt"
         def bundleAssetName = project.extensions.react.bundleAssetName.get() ?: "index.android.bundle"
+        def targetName = variant.name.capitalize()
```

Comment:
`targetName` is the same for all cases, but I prefer to have it defined in here to have a more consistent scope in each `if` condition.
But I'm not against moving it to the top of the `getReactConfig` function if that makes more sense.